src/index.css

```diff
@@ -0,0 +1,34 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+/* global css */
+
+:root {
```

Comment:
````
--bg-gray: #242424;
````
then:

````
background-color: var(--bg-gray);
````

It will allow us to use anywhere `var(--bg-gray)` if we need pure css classes.